src/commands/redis_cmd.cc

```diff
@@ -2380,12 +2381,20 @@ class CommandSInterStore : public Commander {
 class CommandZAdd : public Commander {
  public:
   Status Parse(const std::vector<std::string> &args) override {
-    if (args.size() % 2 != 0) {
-      return Status(Status::RedisParseErr, errInvalidSyntax);
+    unsigned index = 2;
+    parseOptions(args, index);
```

Comment:
Can use consistent name like `parseFlags`

Reply:
@git-hulk I think that `options` is OK since the `Redis` documentation names them as `ZADD options`. Perhaps the function name should be more specific like `parseAddOptions`. However, this method belongs to the class `CommandZAdd` so it may be clear that they are ZAdd-options. What do you think?

Reply:
Yes, flags or options are OK. I saw that it calls 'options' when parsing but used `validateFlags` when validating, so I think it will be better to keep consistent if possible.

Reply:
@git-hulk Now I see. Yes, you are right here. There should be one name everywhere for them (flags or options).

Reply:
👍 Yes, I didn't claim the context clearly in the first comment.

Reply:
Ok, changed to parseFlags now.